### PR TITLE
Update scalardb-analytics-spark-sample to support 3.14

### PR DESCRIPTION
**Pull request by choplin**
## Description

This PR updates `scalardb-analytics-spark-sample` for the 3.14 release.

## Related issues and/or PRs

N/A

## Changes made

- We now launch PostgreSQL, MySQL and Cassandra. We use ScalarDB to manage MySQL and Cassandra. PostgreSQL is used independently.
  - This is intended to demonstrate the virtualization feature introduced in 3.14.
- Update the depending ScalarDB version to 3.14
- Update Gradle version

## Checklist

- [ ] I have commented my code, particularly in hard-to-understand areas.
- [x] I have updated the documentation to reflect the changes.
- [x] Any remaining open issues linked to this PR are documented and up-to-date (Jira, GitHub, etc.).
- [ ] Tests (unit, integration, etc.) have been added for the changes.
- [x] My changes generate no new warnings.
- [x] Any dependent changes in other PRs have been merged and published.

## Additional notes (optional)

None

### CODE REVIEW

File: scalardb-analytics-spark-sample/sample-data-loader/build.gradle.kts [hunk 1]
```diff
@@ -9,10 +9,11 @@ repositories {
 }
 
 dependencies {
-    implementation("com.scalar-labs:scalardb:3.12.1")
-    implementation("com.scalar-labs:scalardb-schema-loader:3.12.1")
+    implementation("com.scalar-labs:scalardb:3.14.0")
+    implementation("com.scalar-labs:scalardb-schema-loader:3.14.0")
     implementation("org.apache.commons:commons-csv:1.10.0")
 
+
```
Comment (brfrn169) [inline scalardb-analytics-spark-sample/sample-data-loader/build.gradle.kts:16]: Maybe this empty line is unnecessary?

**Review by brfrn169 — APPROVED** [review]
Left a minor comment. Other than that, LGTM! Thank you!

File: scalardb-analytics-spark-sample/sample-data-loader/build.gradle.kts (hunk 1 above)
Comment (choplin) [reply scalardb-analytics-spark-sample/sample-data-loader/build.gradle.kts:16]: Right. I'll remove it. Thanks!

Comment (choplin) [reply scalardb-analytics-spark-sample/sample-data-loader/build.gradle.kts]: Addresses by https://github.com/scalar-labs/scalardb-samples/pull/75/commits/100cb459f3d6cd5d22199aadc428707a9f18a68b

**Comment by choplin** [comment]
BTW, I noticed that many unrelated Actions run. It would be worth setting more strict filtering.

File: scalardb-analytics-spark-sample/docker-compose.yml [hunk 2]
```diff
@@ -55,50 +55,52 @@ services:
       interval: 1s
       timeout: 1s
       retries: 10
-      start_period: 5s
+      start_period: 10s
 
-  backend-dynamodb:
-    image: amazon/dynamodb-local:1.21.0
+  scalardb-mysql:
+    image: mysql:8.0.36
```
Comment (komamitsu) [inline scalardb-analytics-spark-sample/docker-compose.yml:61]: [trivial] How about just `8.0` to use the latest patch version?

File: scalardb-analytics-spark-sample/sql/postgres_copy.sql [hunk 3]
```diff
@@ -0,0 +1,13 @@
+create schema sample_ns;
```
Comment (komamitsu) [inline scalardb-analytics-spark-sample/sql/postgres_copy.sql:1]: Is there any instruction to execute this SQL file?

Comment (choplin) [reply scalardb-analytics-spark-sample/sql/postgres_copy.sql:1]: No. But, the `postgres` image automatically executes any `.sql` and `.sh` in `/docker-entrypoint-initdb.d`, so we just mount this file under that directory.

File: scalardb-analytics-spark-sample/docker-compose.yml (hunk 2 above)
Comment (choplin) [reply scalardb-analytics-spark-sample/docker-compose.yml:61]: Sure. I'll change the version later.

Comment (choplin) [reply scalardb-analytics-spark-sample/docker-compose.yml]: Addressed by https://github.com/scalar-labs/scalardb-samples/pull/75/commits/319f75490c0e3b4d09037a73561762132ac8ab58

**Review by feeblefakie — APPROVED** [review]
LGTM! Thank you!

File: scalardb-analytics-spark-sample/sql/postgres_copy.sql (hunk 3 above)
Comment (komamitsu) [reply scalardb-analytics-spark-sample/sql/postgres_copy.sql:1]: >  the postgres image automatically executes any .sql and .sh in /docker-entrypoint-initdb.d, so we just mount this file under that directory.

Oh, I didn't know that. Sounds good!

**Review by komamitsu — APPROVED** [review]
LGTM! 👍

File: scalardb-analytics-spark-sample/sql/postgres_copy.sql (hunk 3 above)
Comment (choplin) [reply scalardb-analytics-spark-sample/sql/postgres_copy.sql:1]: JFYI, refer to the "Initialization scripts" section on the image description. https://hub.docker.com/_/postgres

AFAIK, several database docker images have similar functionality.

**Pull request merged**